packages/server/routes/clientRouter.ts

```diff
@@ -0,0 +1,13 @@
+import express from "express";
+import clientIndexPath from "../config/clientIndexPath.js";
+
+const clientRouter = express.Router();
+const clientRoutes = ["/"];
+
+clientRouter.get(clientRoutes, (req, res) => {
+  res.sendFile(clientIndexPath(), (err) =>
```

Comment:
If the server is hit at paths specified (we would want to add any `react-router` paths to `clientRoutes`) then return the HTML file featuring the React app injection.